### PR TITLE
libxsmm: init at 1.13

### PR DESCRIPTION
**Pull request by chessai**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @chessai

### CODE REVIEW

File: pkgs/development/libraries/libxsmm/default.nix [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    which
+  ];
+
+  prePatch = ''
+    patchShebangs .
+  '';
+
+  installPhase = ''
+    runHook preInstall
+    make install SHELL=${bash}/bin/bash PREFIX=$out
```
Comment (matthewbauer) [inline pkgs/development/libraries/libxsmm/default.nix]: This should use ```makeFlags = [ "PREFIX=$(out)" ];``` to be more concise. SHELL is provided automatically.

**Comment by doronbehar** [comment]
I'm closing as OP seems not intent to respond to review requests.

**Comment by chessai** [comment]
@doronbehar thanks for the ping! responded to @matthewbauer's review

**Comment by chessai** [comment]
thanks @matthewbauer

**Pull request merged**